packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -384,9 +387,7 @@ export class GarbageCollector implements IGarbageCollector {
 
 		// If the GC state has been initialized, update the tracking of unreferenced nodes as per the current
 		// reference timestamp.
-		for (const [, nodeStateTracker] of this.unreferencedNodesState) {
-			nodeStateTracker.updateTracking(currentReferenceTimestampMs);
-		}
+		this.unreferencedNodesState.updateAllTracking(currentReferenceTimestampMs);
```

Comment:
Similar to my comment for the other helper, this helper doesn't add any value IMO. I like the idea of overriding "delete" to also stop the tracking. Maybe that's all that is needed?

Reply:
Yeah, I can revert it.  I think it's possible that there's a way to encapsulate this tracking more (so, less logic in GC class), but it's pretty much vanity to pursue at this point.  It's fine as-is.